src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -49,10 +49,6 @@
   <!-- Team foundation assemblies -->
   <!-- ******************************************************************************* -->
   <ItemGroup Label="TeamFoundation" Condition="$(RequiresTeamFoundation)=='true'">
-    <Reference Include="Microsoft.TeamFoundation.Client">
-      <HintPath>$(ThirdPartyPath)\Microsoft.TeamFoundation.Client.dll</HintPath>
-      <Private>$(CopyVSAssembliesToOutput)</Private>
-    </Reference>
     <Reference Include="Microsoft.TeamFoundation.Controls">
```

Comment:
This is still needed to show a button in the TE ui, right? Is there a ticket to eventually remove this too?

Reply:
https://sonarsource.atlassian.net/browse/SLVS-1515